### PR TITLE
BWDB-11530 Updated Create & Update Brand endpoints to match with TCR

### PR DESCRIPTION
**Pull request by AswiniKumarV**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-bwdb-11530.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-bwdb-11530.s3-website-us-east-1.amazonaws.com/

File: campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md [hunk 1]
```diff
@@ -620,14 +623,15 @@ _Note_: Non-editable fields will be ignored and will not be updated.
...
+| `PostalCode`               | Yes       | Yes      | Postal codes. Use 5 digit zipcode for United States  |
 | `Country`                  | Yes       | Yes      | ISO2 2 characters country code. Example: US - United States   |
 | `Email`                    | Yes       | Yes      | Valid email address of brand support contact. Max 100 characters   |
-| `Phone`                    | No        | Yes      | Valid phone number in e.164 international format '+18009999999' |
-| `Vertical`                 | Yes       | Yes      | Enum value describing vertical or industry segment of the brand   |
+| `Phone`                    | Yes       | Yes      | Valid phone number in e.164 international format '+18009999999' |
+| `Vertical`                 | Yes (Not required for Sole_Proprietor)       | Yes      | Enum value describing vertical or industry segment of the brand   |
+| `BrandRelationship`        | Yes       | Yes      | Enum value describing the relationship with your Account. Supported values: 'BASIC_ACCOUNT', 'SMALL_ACCOUNT', 'MEDIUM_ACCOUNT', 'LARGE_ACCOUNT', 'KEY_ACCOUNT' |
```
Comment (jhough07) [inline campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md:634]: I don't think BrandRelationship is editable on the PUT

Comment (AswiniKumarV) [reply campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md:634]: @jhough07 Thanks, I'll update. Just wanted to check: But it is still part of the request and response but not editable, right?

Comment (AswiniKumarV) [reply campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md:634]: A follow-up question. What if the reseller has to update `brandRelationship` for a customer brand after a few months? Can that even be possible.

Comment (jhough07) [reply campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md:634]: I was incorrect, it was editable

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-bwdb-11530.s3-website-us-east-1.amazonaws.com/

**Review by jhough07 — APPROVED** [review]

**Review by jmulford-bw — APPROVED** [review]

**Pull request merged**